act/utils/io_utils.py

```diff
@@ -376,3 +380,190 @@ def generate_movie(images, write_filename=None, fps=10, **kwargs):
         clip.write_videofile(str(write_filename), **kwargs)
 
     return str(write_filename)
+
+
+def arm_standards_validator(file=None, dataset=None, verbose=True):
+    """
+    ARM Data Validator (ADV) - Checks to ensure that ARM standards are being followed
+    in the files or dataset passed to it.  Note, this includes a minimal set of
+    standards that it checks against
+
+    ...
```

Comment:
Can we remove this line and keep one space between the intro and parameter header

Reply:
Oh good catch haha! Thanks for doing that!